openfisca_france_local/departements/eure_et_loir/ASH.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from openfisca_france.model.base import Variable, Individu, Menage, MONTH
+
+class eure_et_loir_ASH_personne_agee(Variable):
+    value_type = bool
+    entity = Individu
+    definition_period = MONTH
+    label = "Éligibilité d'une personne agée à l'aide sociale à l'hébergement en établisement"
+    reference = """Titre 2 Chapitre 2-1 du Règlement départemental d'Aide Sociale PA PH de l'Eure et Loir
+                   L’aide sociale à l’hébergement en établissement s’adresse aux personnes résidant en EHPAD, en résidence autonomie ou en USLD et dont les ressources ne permettent pas de faire face aux frais d’hébergement. 
+                   Les ressources de la personne (hors prestations familiales) sont récupérées dans la limite de 90% (AL et APL récupérées à 100%).
+                   Cette aide fait l’objet d’une récupération sur succession.
+                """
+
+    def formula_2020_01(individu,period,parameters):
+        age = individu('age', period)
+        inapte_travail = individu('inapte_travail', period)
+        ressortissant_eee = individu('ressortissant_eee', period)
+        duree_possession_titre_sejour = individu('duree_possession_titre_sejour', period)
+        individual_resource_names = {
+            'ass',
+            'chomage_net',
+            'retraite_nette',
+            'salaire_net',
+            'allocation_securisation_professionnelle',
+            'dedommagement_victime_amiante',
+            'gains_exceptionnels',
+            'indemnites_chomage_partiel',
+            'indemnites_journalieres',
+            'indemnites_volontariat',
+            'pensions_invalidite',
+            'prestation_compensatoire',
+            'prime_forfaitaire_mensuelle_reprise_activite',
+            'retraite_brute',
+            'revenus_stage_formation_pro',
+            'rsa_base_ressources_patrimoine_individu'
+        }
+
+        individu_resources = sum([individu(resource, period.last_month) for resource in individual_resource_names])
+
+        condition_age = ((age >= parameters(
+            period).departements.eure_et_loir.ASH.age_minimal_personne_agee_apte_travail) or (
+                                     age >= parameters(
+                                 period).departements.eure_et_loir.ASH.age_minimal_personne_agee_inapte_travail and inapte_travail))
```

Comment:
Employer les [opérateurs compatibles avec le calcul vectoriel](https://openfisca.org/doc/coding-the-legislation/25_vectorial_computing.html#boolean-operations)? 
Factoriser le parcours de l'arbre des paramètres commun ? 
Et indenter en fonction des opérations ? 
> Le cas inapte_travail est-il bien parenthésé ? Sera vérifié à la revue des tests :) 
```suggestion
        ash_parameters = parameters(period).departements.eure_et_loir.ASH
        condition_age = (
            (age >= ash_parameters.age_minimal_personne_agee_apte_travail)
            + (age >= ash_parameters.age_minimal_personne_agee_inapte_travail * inapte_travail)
            )
```